pkg/api/v1alpha1/authenticated_user.go

```diff
@@ -73,7 +74,7 @@ func (r *Router) getAuthenticatedUser(c *gin.Context) {
 
 	if ctxUser.R == nil {
 		c.JSON(http.StatusOK, AuthenticatedUser{
-			User:  &User{ctxUser, []string{}, []string{}},
+			User:  &User{ctxUser, []string{}, []string{}, []string{}},
```

Comment:
I know you didn't write this whole line, but we probably want to use named fields here. In general initializing a struct like this risks things breaking if fields get reordered.